src/components/scenes/FormScene.tsx

```diff
@@ -17,15 +18,13 @@ interface OwnProps {
 type Props = OwnProps
 
 export const FormScene = (props: Props) => {
-  const { headerText, children, onSliderComplete, sliderDisabled } = props
+  const { headerText, headerIcon, children, onSliderComplete, sliderDisabled } = props
   const theme = useTheme()
   const styles = getStyles(theme)
 
-  const sceneHeader = React.useMemo(() => <SceneHeader style={styles.sceneHeader} title={headerText} underline />, [headerText, styles.sceneHeader])
-
   return (
     <SceneWrapper background="theme">
-      {sceneHeader}
+      <SceneHeader style={styles.sceneHeader} title={headerText} icon={headerIcon} withTopMargin underline />
```

Comment:
Why was withTopMargin added?

Reply:
Because that's what is "looks like" from the designs.